docs/resources/azurerm_mysql_databases.md.erb

```diff
@@ -0,0 +1,115 @@
+---
+title: About the azurerm_mysql_databases Resource
+platform: azure
+---
+
+# azurerm\_mysql\_databases
+
+Use the `azurerm_mysql_databases` InSpec audit resource to test properties and configuration of Azure MySQL Databases.
+<br />
+
+## Azure REST API version
+
+This resource interacts with version `2017-12-01` of the Azure Management API. For more
+information see the [Official Azure Documentation](https://docs.microsoft.com/en-us/rest/api/mysql/databases/listbyserver).
+
+At the moment, there doesn't appear to be a way to select the version of the
+Azure API docs. If you notice a newer version being referenced in the official
+documentation please open an issue or submit a pull request using the updated
+version.
+
+## Availability
+
+### Installation
+
+This resource is available in the `inspec-azure` [resource
+pack](https://www.inspec.io/docs/reference/glossary/#resource-pack). To use it, add the
+following to your `inspec.yml` in your top-level profile:
+
+    depends:
+      inspec-azure:
+        git: https://github.com/inspec/inspec-azure.git
+
+You'll also need to setup your Azure credentials; see the resource pack
+[README](https://github.com/inspec/inspec-azure#inspec-for-azure).
+
+### Version
+
+This resource first became available in 1.6.0 of the inspec-azure resource pack.
+
+## Syntax
+
+An `azurerm_mysql_databases` resource block returns all MySQL Databases on a MySQL Server, within a Rsource Group.
+
+    describe azurerm_mysql_databases(resource_group: ..., server_name: ...) do
+      ...
+    end
+
+<br />
+
+## Examples
+
+The following examples show how to use this InSpec audit resource.
+
+### Check MySQL Databases are present
+
+    describe azurerm_mysql_databases(resource_group: 'resource-group-1', server_name: 'production') do
+      it            { should exist }
```

Comment:
This probably isn't an issue with this PR, but I was wondering whether there is a need for `exist` in plural resources. I believe the same thing could be achieved through `it { should_not be_empty }`.